packages/renderer/src/lib/engine/EngineInfoUI.ts

```diff
@@ -20,5 +20,5 @@
 export interface EngineInfoUI {
   id: string;
   name: string;
-  type: 'podman' | 'docker';
+  type?: 'podman' | 'docker';
```

Comment:
I guess it should be kept as mandatory

Reply:
Yeah, this is leftover from the previous PR. We should actually remove this completely / we don't need the type.